projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -5,6 +5,41 @@ html { box-sizing: border-box; }
 body {
 	font-family: sans-serif;
 	font-size: 14px;
+	font-weight: bold;
 }
 
+.container {
+	display: flex;
+	background-color: #DCDCDC;
+	border-radius: 10px;
+	margin: 1rem;
+	padding: 1rem;
+}
+
+.container2 {
+	display: flex;
+	display: table-column;
+	width: 22%;
+	background-color: white;
+	margin: 1rem;
+	padding: 1rem;
+	text-align: center;
+	border-radius: 10px;
+	flex-grow: 1;
+}
+
+.foto {
+	width: 150px;
+	height: 150px;

```

Comment:
Výška zde vyloženě nutná není, pokud ji neměníš dopočítá se podle šířky (a naopak), tady to nevadí, ale hodí se to pro vyhnutí se deformacím :)).